reporters_db/data/reporters.json

```diff
@@ -17996,6 +18069,21 @@
             "variations": {}
         }
     ],
+    "Quarto": [
+        {
+            "cite_type": "specialty",
+            "editions": {
+                "Quarto": {
+                    "end": "1851-12-31T00:00:00",
+                    "start": "1850-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [],
+            "name": "Howison's Criminal Trials",
```

Comment:
is this the correct name

Reply:
I think is correct: https://www.legalabbrevs.cardiff.ac.uk/titles/tview/id/9298/tsearch//ttype/Exact

Reply:
How do we get to the name Quarto?

Reply:
That was my mistake, the correct is howison. I'll correct it.